exercises/java-primitives/src/main/java/com/cbfacademy/StringExercises.java

```diff
@@ -3,19 +3,68 @@
 public class StringExercises {
 
     public String fromCharacters(char[] characters) {
-        // TODO - Write code that creates a String from the input array of characters
-        throw new RuntimeException("Not implemented");
+        //TODO - Write code that creates a String from the input array of characters
+
+        String sentence = new String (characters);
+        return sentence;
+        
     }
 
     public Long howMany(String text, Character character) {
         // TODO - Write code to determine how many of the input ${character} are contained in the input ${text}
-        throw new RuntimeException("Not implemented");
+
+        if (text == null || text.isEmpty() || character == null) {
+            return 0L;
+        }
+    
+        long count = 0L;
```

Comment:
If you define this variable before your if statement, you can just return `count` if text is null, etc.